packages/time/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Timestamps, time math, timer service API definition",
   "type": "module",
   "main": "index.js",
+  "types": "index.js",
```

Comment:
attn @mhofman This fixed a bunch of type imports. The working alternative is to use a deep import to `import('@agoric/time/src/types.js')` which is not good™. Is this controversial?

Reply:
Ugh I don't understand why this is needed. Can you explain why tsc doesn't use the types from the main export?

Reply:
Aka why, without this line, is `import('@agoric/time').Foo` not sufficient?

Reply:
I can offer no explanation. This should be orthogonal to the endo integration, but something leaked somehow.

Reply:
In this case it looks fine, but have you checked that `import('@agoric/time').RelativeTime` doesn't resolve to `any` ?

Reply:
Out of band, we learned that the namespace object had the any type, but individual exports were well-defined. We also learned that this works because `@agoric/time/index.js` is coïncidentally a valid `.d.ts` file.